.rancher-pipeline.yml

```diff
@@ -17,18 +17,18 @@ stages:
   - publishImageConfig:
       dockerfilePath: ./Dockerfile
       buildContext: .
-      tag: docker.dev.folio.org/mod-source-record-storage:folijet-${CICD_EXECUTION_SEQUENCE}
+      tag: docker.dev.folio.org/mod-source-record-storage:spitfire-${CICD_EXECUTION_SEQUENCE}
```

Comment:
please don't change folijet project in master

Reply:
Sure